src/main/java/com/salesforce/mirus/MirusSourceTask.java

```diff
@@ -201,12 +229,12 @@ private SourceRecord toSourceRecord(ConsumerRecord<byte[], byte[]> consumerRecor
     return new SourceRecord(
         sourcePartition,
         offsetMap(consumerRecord.offset() + 1),
-        destinationTopicNamePrefix + consumerRecord.topic() + destinationTopicNameSuffix,
+        topic,
         enablePartitionMatching ? consumerRecord.partition() : null,
-        Schema.OPTIONAL_BYTES_SCHEMA,
-        consumerRecord.key(),
-        Schema.OPTIONAL_BYTES_SCHEMA,
-        consumerRecord.value(),
+        keySchema,
```

Comment:
You can simplify this change by calling `keyAndSchema.schema()`,  `keyAndSchema.value()`, etc. directly here.  That will allow you to remove the local variable definitions above.